package.json

```diff
@@ -46,11 +46,17 @@
     "lodash": "^4.17.15"
   },
   "resolutions": {
```

Comment:
There's a lot of new resolutions here. Are we sure these are the right versions? Looking at the yarn lock file it looks like some new/different versions being installed. I'd rather separate "passing yarn check" from "upgrading the version of dependencies resolved".

Reply:
I used the latest version used somewhere else in our source code. We must be consistent.
Anyway, the change in the `package.json` file is reverted.